app/controllers/api/v1/bulk_download_controller.rb

```diff
@@ -106,13 +138,29 @@ def summary
         render json: @study_file_info
       end
 
+      def drs_info
```

Comment:
A comment on why this DRS information is needed, and how it's used, would help here.

Reply:
I'll actually add it to the swagger docs for consistency since it is a public method.